blueprints/blog.py

```diff
@@ -0,0 +1,55 @@
+# All Blog endpoints
+from flask import Blueprint, request, jsonify
+from db import orm
+from models import Blog
+
+blog_bp = Blueprint('blog_bp', __name__)
```

Comment:
```suggestion
blog_bp = Blueprint('blog_bp', __name__,  url_prefix="/blogs")
```

You can use `url_prefix` parameter to `Blueprint` class. It will add **"/blogs"** to all other endpoints which were created using `@blog_bp` automatically. Now you can remove the **"/blogs"** from each route. 

Endpoints then look like:
- `@blog_bp.route('/', methods=['GET'])`
- `@blog_bp.route('/', methods=['POST'])`
- `@blog_bp.route('/<int:blog_id>', methods=['PUT'])`
- `@blog_bp.route('/<int:blog_id>', methods=['DELETE'])`

Reply:
done